deeppavlov/core/agent/agent.py

```diff
@@ -51,7 +52,7 @@ class Agent(Component, metaclass=ABCMeta):
         dialog_logger: DeepPavlov dialog logging facility.
     """
     def __init__(self, skills: List[Skill]) -> None:
-        self.skills: List[Skill] = skills
+        self.skills: Union[List[Skill], List[Component]] = skills
```

Comment:
Missed this change in the method definition

Reply:
fixed